src/Backend/Core/Engine/Model.php

```diff
@@ -707,7 +707,7 @@ public static function insertExtra(
                 'module' => $module,
                 'type' => $type,
                 'label' => $label ?? $module, // if label is empty, fallback to module
-                'action' => $action ?? $module, // if action is empty, fallback to module
+                'action' => $action ?? null,
```

Comment:
Am I wrong to think  that this is then also an issue in the current master branch?
https://github.com/forkcms/forkcms/blob/master/src/Backend/Core/Engine/Model.php#L747